test/mocks/Battle.spec.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const assert = require('../assert');
+const Battle = require('./Battle');
+
+describe('Battle', () => {
+	it('should reset the PRNG to original when I call resetRandomNumberGenerator', () => {
```

Comment:
@Slayer95  Sure. Unit testing is not my strongest suit. How does:

`should reset the PRNG to the original when resetRNG is called`

sound?